numba/core/removerefctpass.py

```diff
@@ -0,0 +1,123 @@
+"""
+Implement a rewrite pass on LLVM module to remove unnecessary refcount
+operation.
+"""
+
+from llvmlite.ir.transforms import CallVisitor
+
+from numba.core import types
+
+
+class _MarkNrtCallVisitor(CallVisitor):
+    """
+    A pass to mark all NRT_incref and NRT_decref.
+    """
+    def __init__(self):
+        self.marked = set()
+
+    def visit_Call(self, instr):
+        if getattr(instr.callee, 'name', '') in ('NRT_incref', 'NRT_decref'):
```

Comment:
use this https://github.com/numba/numba/pull/5795/files#diff-6d3d7768c0d85266cac26db329c495bfR35 in the `contains`?